### PR TITLE
Hello Microverse text

### PR DESCRIPTION
**Pull request by ishimwezachee**
In this project, I done the following 
- hello mircroverse text using HTML and CSS 
- added linters in the project
- added gitignore file 
- created a new branch from main to merged

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
```
Comment (carlos-ssh) [inline .github/workflows/linters.yml:48]: - Check details in the linter error displayed in your pull request and fix it.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+/test.md
```
Comment (carlos-ssh) [inline .gitignore]: - your project it's missing node_modules.

_Check your project requirements and implement the requested installations.. could be consider to have installed linters from npm._

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @ishimwezachee , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 2 above)
Comment (ishimwezachee) [reply .gitignore]: Hello carlos-ssh, I have managed to fix what you've asked me to, but when I run the npx stylelint ., I am getting an error called unknown word CssSyntaxError, any ideas of how I can pass it ?

File: README.md [hunk 3]
```diff
@@ -1 +1,41 @@
-# hello-microverse
\ No newline at end of file
+
+# Hello microverse
+
+> This is my first project at microverse to practice gihub workflow 
+
+
+
+It just display a simple hello microverse text
```
Comment (DanteAlonsoHT) [inline README.md]: - [x] [OPTIONAL] For the next projects you should add a better description, just try to explain what you did, what is the main scope, what kind of technologies or techniques you used, and so on. (Add like 3 lines or more at least in your description from your README file for the next time 😉).

**Review by DanteAlonsoHT — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @ishimwezachee 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**